libraries/plugins/json_rpc/json_rpc_plugin.cpp

```diff
@@ -227,6 +232,12 @@ namespace detail
       {
          response.error = json_rpc_error( JSON_RPC_INVALID_REQUEST, "jsonrpc value is not \"2.0\"" );
       }
+
+      idump( (request) );
```

Comment:
Let's make these debug level logging (`ddump`). Our nodes are set up for info level logging, and this is a bit much for our logging infrastructure to save.

Reply:
Done.